### PR TITLE
#8:  Allow multiple selection

### PR DESCRIPTION
**Pull request by alvinosh**
- #8
- CI build for macOS

### CODE REVIEW

File: src/routes/+page.svelte [hunk 1]
```diff
@@ -70,12 +73,7 @@
 		}
 	}
 
-	$: if ($galleryStore.selectedTag) {
```
Comment (prokawsar) [inline src/routes/+page.svelte]: @alvinosh 
This part of the code works for search with tags. 
I think it isn't related to folder selection on Linux issues.

File: src/routes/+page.svelte [hunk 2]
```diff
@@ -5,51 +5,54 @@
...
-	import { toast } from 'svelte-sonner'
-
-	let images: ImageType[] = []
-	let zoomLevel: number[] = [$galleryStore.zoomLevel]
 
 	const uploadFolder = async () => {
-		images = []
-		$galleryStore.images = []
-		const filesDir: File[] = (await openDirectory()) as File[]
+		let selected = await open({
```
Comment (prokawsar) [inline src/routes/+page.svelte]: I tested the app on the browser, it doesn't open the dialog on the browser, will test it on the Tauri build app and update the comment.

Comment (prokawsar) [reply src/routes/+page.svelte]: @alvinosh 
Tested on Mac too, It doesn't work on the browser.
And for the Tauri app, it seems it works for only file selection, not for the whole folder.

cc: @kirillt

File: src/routes/+page.svelte (hunk 1 above)
Comment (prokawsar) [reply src/routes/+page.svelte]: I updated this part myself.

File: src/routes/+page.svelte (hunk 2 above)
Comment (kirillt) [reply src/routes/+page.svelte]: @prokawsar Functioning in browser is less priority than in Tauri. Convenient for development, but we need only standalone app.

@alvinosh In Linux, I can select multiple files (cannot folder), but then nothing happens. Images don't load.

Comment (alvinosh) [reply src/routes/+page.svelte]: @prokawsar Im pretty sure when working with tauri the only reason there is a localhost running is for hot reloading duirng development purposes, you are supposed to do all your testing in the desktop application.

Comment (alvinosh) [reply src/routes/+page.svelte]: @kirillt Folder not being able to be selected is expected behaviour (the allowFolder option is not set accidentally). But images should load, is there any errors in the console?

**Review by kirillt — APPROVED** [review]
* Works on macOS
* Works on Windows
* 50/50 on Linux (my Fedora with only tiled WM is not typical environment)

Thanks, that's a good job!

**Pull request merged**